README.md

```diff
@@ -9,7 +9,7 @@ This is the CMS for the University of Alberta Libraries website. It uses Comfy M
 ## Development environment
 
 + Ruby 3.1.4
-+ Rails 7.0.2
++ Rails 7.0.6
```

Comment:
```suggestion
+ Rails 7.0.x
```
Perhaps if we only capture the minor version this will need to change less often?

Reply:
> 

Makes sense, would make things easier down the road